CAP/gap/UniversalObjects.gi

```diff
@@ -276,6 +276,73 @@ InstallMethod( DirectProductFunctorial,
     
 end );
 
+##
```

Comment:
The UniversalMorphism.gi file should only contain
convenient methods, and not methods that look like complete derivations.


I suggest to define `AssociatorRightToLeftOfDirectProductsWithGivenDirectProducts`
as a new CAP basic operation, and to install this particular method
as a derivation. The advantage is that `AssociatorRightToLeftOfDirectProductsWithGivenDirectProducts`
would then only be installed for categories that already can compute
 `ProjectionInFactorOfDirectProductWithGivenDirectProduct`,
`UniversalMorphismIntoDirectProductWithGivenDirectProduct`, etc.